config/config.js.sample

```diff
@@ -3,97 +3,97 @@
  * By Michael Teeuw http://michaelteeuw.nl
  * MIT Licensed.
  *
- * For more information how you can configurate this file
+ * For more information how you can configuration this file
```

Comment:
This doesn't seem correct.

Reply:
I completely butchered the Queen's language. Will fix it on next commit.